.devcontainer/devcontainer.json

```diff
@@ -27,24 +29,32 @@
                 },
                 "isort.args": ["--profile", "black"],
                 "python.analysis.autoImportCompletions": true,
-                "python.defaultInterpreterPath": "/usr/local/conda/envs/Recommenders/bin/python",
+                // Conda env name *must* align with ENV_HOME in Dockerfle
```

Comment:
I think, as Miguel suggested, that it would be good to have the instructions in a SETUP page somewhere as well.

Reply:
Sure